### PR TITLE
Little doc improvement due to issue #22566

### PR DESCRIPTION
**Pull request by jjlopezm**
This PR add to doc two params that are neccessary to start an akka cluster using remoting with ssl.
The related issue is :  https://github.com/akka/akka/issues/22566

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @jjlopezm,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by jjlopezm** [comment]
I just sign the CLA :)

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

File: akka-docs/rst/java/remoting.rst [hunk 1]
```diff
@@ -450,6 +450,9 @@ Next the actual SSL/TLS parameters have to be configured::
   akka {
     remote {
       netty.ssl.security {
+        hostname = "127.0.0.1"
```
Comment (patriknw) [inline akka-docs/rst/java/remoting.rst]: I'm not sure this is correct. Shouldn't it be in the `netty.ssl {` section?

**Review by patriknw — CHANGES_REQUESTED** [review]

File: akka-docs/rst/java/remoting.rst (hunk 1 above)
Comment (jjlopezm) [inline akka-docs/rst/java/remoting.rst]: You're totally right. I'm going to change it.

**Comment by raboof** [comment]
LGTM (but needs conflict resolution)

**Comment by ktoso** [comment]
Resolved the conflicts, was very simple. LGTM

**Review by ktoso — APPROVED** [review]

**Comment by jjlopezm** [comment]
Thanks! 👍

**Comment by raboof** [comment]
Thank YOU! :)

**Pull request merged**